script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -878,46 +888,46 @@ In the middle of it all―――
 [sha:5936ee14ab501c4927ddc86ebc341ac4d7060768]{
 -- Page 710, Offset 10583.
 -- ゆらりと、吸血鬼は独白した。
-The vampire muses to himself as he sways.
+The vampire calmly muses to himself.
 }
 [sha:4c5346adff8f8c3139a3e221711fe60d0b3beaf6]{
 -- Page 711, Offset 10584.
 -- 「“……この傷は知らない。この喪失は思い出せない。
-"I do not know this wound... I cannot remember this loss.
+"I'm not familiar with a wound like this... I cannot recall loss like this.
 }
 [sha:505c740e7a8408848b96a465167c053debc73047]{
 -- Page 711, Offset 10585.
 -- この―――言語化できない大気の肌触りは、いつか失った何物かだ”」
-This―――indescribable feeling of the air on my skin...
- is something I lost some time ago."
+This... indescribable feeling of the air on my skin...
+ is something I once thought lost..."
 }
 [sha:b83a9b5c6f1d97c446abb7657176e51abac4ee1f]{
 -- Page 712, Offset 10586.
 -- 何を言っているのか、
-I don't know what he's talking about―――
+I don't know what he's saying―――
 }
 [sha:52e209242e6d256f000d320108fb8b613022dc53]{
 -- Page 712, Offset 10587.
 -- どこの国の言葉かも分からない。
-Or which country's language he's speaking in.
+Or which language he's speaking in.
 }
 [sha:52fce6f3e316f69d01cc8f964cee06bafc074fde]{
 -- Page 712, Offset 10588.
 -- ただ、穏やかなその声は、春を<謳|うた>う詩人のようだった。
-But his gentle voice calls to mind a poet celebrating the spring.
+But his gentle voice calls to mind a poet celebrating the joys of spring.
 }
 [sha:69a97d8f5e9484360f6aa09be52052768358fd6b]{
 -- Page 713, Offset 10589.
 -- 「“ああ―――痛みも、寒波も感じない。
-"Ah―――I feel no pain, nor do I feel the cold waves.
+"Ahh―――I feel neither the pain, nor the cold waves.
 }
 [sha:a55a4dd1a3e4b5e062f0906848eaf9520c7322fb]{
 -- Page 713, Offset 10590.
 -- 恐ろしいが、懐かしい。
-It is terrifying but familiar.
+It is terrifying, yet familiar.
 }
 [sha:9801e9d84c9d4b68611f826711fd8ad876460399]{
 -- Page 713, Offset 10591.
 -- これではまるで、僕は死人になったようだ”」
-Almost as though I have become a dead man."
+It's almost as if... I've turned dead."
```

Comment:
the old version wasn't fantastic, but i do not like this one at all. "turned dead" just doesn't work.

Reply:
Also prefer original, or maybe
> Almost as though I were a dead man walking.

Reply:
I think dead man walking has entirely different connotations.

The simple construction is still something I think will serve this sentence better.

Maybe just 'I've turned dead' > 'I were dead'